lib/openhab/dsl/rules/automation_rule.rb

```diff
@@ -110,8 +110,10 @@ def create_queue(inputs)
         def extract_event(inputs)
           event = inputs&.dig("event")
           unless event
-            event = Struct.new(:event, :attachment, :command).new
-            event.command = inputs&.dig("command")
+            input_keys = inputs.to_h.keys.grep(/^[a-z_]+[a-z0-9_]*$/).grep_v("module") # only pick valid identifiers
+            event_members = %i[event attachment] | input_keys.map(&:to_sym)
```

Comment:
does `event` need to be part of this struct?

Reply:
good question. I didn't investigate because it was there from the previous code

Reply:
I'll remove it